python/src/trezorlib/debuglink.py

```diff
@@ -599,28 +651,30 @@ def _verify_responses(cls, expected, actual):
                 output.append(textwrap.indent(protobuf.format_message(act), "    "))
                 raise AssertionError("\n".join(output))
 
-    def mnemonic_callback(self, _):
+    def mnemonic_callback(self, _) -> str:
         word, pos = self.debug.read_recovery_word()
         if word != "":
+            assert word is not None
             return word
         if pos != 0:
+            assert pos is not None
             return self.mnemonic[pos - 1]
 
         raise RuntimeError("Unexpected call")
 
 
-@expect(messages.Success, field="message")
+@expect(messages.Success, field="message", ret_type=str)
 def load_device(
-    client,
-    mnemonic,
-    pin,
-    passphrase_protection,
-    label,
-    language="en-US",
-    skip_checksum=False,
-    needs_backup=False,
-    no_backup=False,
-):
+    client: "TrezorClient",
+    mnemonic: Union[List[str], Tuple[str, ...], str],
```

Comment:
let's try this:
`Union[str, Iterable[str]]`

and then change the check below to `if isinstance(mnemonic, str)`

(i have a vague memory that `str` is actually an instance of `Iterable[str]`, but that doesn't matter too much)

Reply:
Done in b70acaf